patches/error+7.0.2.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/node_modules/error/typed.js b/node_modules/error/typed.js
+index fe9effd..0aa0c56 100644
+--- a/node_modules/error/typed.js
++++ b/node_modules/error/typed.js
+@@ -21,8 +21,11 @@ function TypedError(args) {
+         var errorName = camelCase(args.type) + 'Error';
+         args.name = errorName[0].toUpperCase() + errorName.substr(1);
+     }
++    //extend runs in strict mode under lavamoat, causing failed assignment to .name
++    const modifiedArgs = Object.assign({}, args)
++    delete modifiedArgs.name
+     
+-    extend(createError, args);
++    extend(createError, modifiedArgs);
```

Comment:
Looks like this was patched in `errors@7.2.0`, so you should be able to surgically upgrade this dependency.

Reply:
(🙌 @raynos)

Reply:
Cool! updated

Reply:
@kriskowal `error@7` is a blast from the past >_>
I rewrote the entire thing to make it less dynamically magical.